src/app/login/loginForm/LoginForm.tsx

```diff
@@ -0,0 +1,69 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import { useForm } from 'react-hook-form';
+
+import { styles } from './LoginForm.styles';
+import { useLocale } from 'hooks';
+import { LoginFields, LoginFormProps } from './LoginForm.types';
+import { InputField } from 'form/fields/inputField/InputField';
+import { emailValidation } from 'form/validators/emailValidation/emailValidation';
+import { passwordValidation } from 'form/validators/passwordValidation/passwordValidation';
+import { CheckboxField } from 'form/fields/checkboxField/CheckboxField';
```

Comment:
Jakbyś dorzucił dwa barrel file (jeden w `form/fields`, a drugi w `form/validators`) to zamiast czterech importów dałoby się zrobić dwa ;)